rolling/distribution.yaml

```
@@ -7089,6 +7089,12 @@ repositories:
       url: https://github.com/ros2/rosidl_runtime_py.git
       version: rolling
     status: maintained
+  rosidl_rust:
```

There's not very much documentation in this package yet, but there probably will be. Any reason not to add a `doc` job for this package?

```suggestion
  rosidl_rust:
    doc:
      type: git
      url: https://github.com/ros2-rust/rosidl_rust.git
      version: main
```